docs/modules/analytic/analytic.rst

```diff
@@ -0,0 +1,7 @@
+pyrato.analytic.analytic
```

Comment:
delete,
we dont need this level of detail, this subdevision of the analytic model is just internally. From extern you still call all funtions from analytic. e.g. ``pyrato.analytic.rectangular_room_rigid_walls``  which is in analytic.py or ``pyrato.analytic.rectangular_room_impedance`` from impedance.py. The documentaiton should habe the same style